dependencies.list

```diff
@@ -1,5 +1,5 @@
 PACKAGE_NAME=realm-js
 VERSION=6.0.3
-REALM_CORE_VERSION=6.0.9
-REALM_SYNC_VERSION=5.0.8
+REALM_CORE_VERSION=6.0.10
```

Comment:
Any reason not to go all the way to `6.0.14`?
(https://github.com/realm/realm-core/tree/v6.0.14)

Reply:
no we should. it was not announced thought